rapid7_insightvm_cloud/icon_rapid7_insightvm_cloud/connection/connection.py

```diff
@@ -0,0 +1,26 @@
+import insightconnect_plugin_runtime
+from .schema import ConnectionSchema, Input
+
+# Custom imports below
+from requests import Session
+from icon_rapid7_insightvm_cloud.util.api import IVM_Cloud
+
+
+class Connection(insightconnect_plugin_runtime.Connection):
+    def __init__(self):
+        super(self.__class__, self).__init__(input=ConnectionSchema())
+        self.ivm_cloud_api = None
+        self.api_url = None
+        self.session = None
+
+    def connect(self, params):
+        self.logger.info("Connect: Connecting...")
+        self.api_url = "https://" + params.get(Input.REGION) + ".api.insight.rapid7.com/vm/v4/integration/"
+        self.session = Session()
+        self.ivm_cloud_api = IVM_Cloud(params.get(Input.CREDENTIALS).get("secretKey"), self.logger, self.api_url)
+
+    def test(self):
+        regions = ["us", "eu", "ca", "au", "ap"]
```

Comment:
Maybe we should just test the region for which the connection is configured?

Reply:
Tests do not take params as far as I can tell, so it's better to check all possibilities.

Reply:
The test will be unique to the connction. so you should be able to do set a `self.region` above on initialization and then use it again here

Reply:
or set api_url to just the base and then change the bath for the actions in question. Couple different ways to do it but we'll want to make sure it only tests for the connection set up since it will fail for others (or should)